examples/benchmarks/arewefastyet/nbody.effekt

```diff
@@ -0,0 +1,179 @@
+import io/args
+import text/string
+import mutable/array
+
+val PI = 3.141592653589793
+val SOLAR_MASS = 4.0 * PI * PI
+val DAYS_PER_YER = 365.24
+
+record Body(
+  x: Double,
+  y: Double,
+  z: Double,
+  vx: Double,
+  vy: Double,
+  vz: Double,
+  mass: Double
+)
+
+def makeBody(
+    x: Double,
+    y: Double,
+    z: Double,
+    vx: Double,
+    vy: Double,
+    vz: Double,
+    mass: Double): Body =
+  Body(x, y, z, vx * DAYS_PER_YER, vy * DAYS_PER_YER, vz * DAYS_PER_YER, mass * SOLAR_MASS)
+
+val jupiter = makeBody(
+  4.84143144246472090,
+  -1.16032004402742839,
+  -0.10362204447112311,
+  0.001660076642744037,
+  0.007699011184197404,
+  -0.0000690460016972063,
+  0.0009547919384243266
+)
+
+val saturn = makeBody(
+  8.34336671824458,
+  4.124798564124305,
+  -0.4035234171143214,
+  -0.002767425107268624,
+  0.004998528012349172,
+  0.00002304172975737639,
+  0.0002858859806661308
+)
+
+val uranus = makeBody(
+  12.89436956213913,
+  -15.11115140169863,
+  -0.22330757889265574,
+  0.002964601375647616,
+  0.0023784717395948095,
+  -0.000029658956854023755,
+  0.00004366244043351563
+)
+
+val neptune = makeBody(
+  15.379697114850917,
+  -25.919314609987964,
+  0.17925877295037118,
+  0.002680677724903893,
+  0.001628241700382423,
+  -0.00009515922545197159,
+  0.00005151389020466115
+)
+
+val sun = makeBody(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0)
+
+def run(n: Int) = {
+
+  def createBodies(): Array[Body] = {
+    val bodies: Array[Body] = emptyArray[Body](5)
+
+    bodies.put(0, sun);
+    bodies.put(1, jupiter);
+    bodies.put(2, saturn);
+    bodies.put(3, uranus);
+    bodies.put(4, neptune);
+
+    var px = 0.0;
+    var py = 0.0;
+    var pz = 0.0;
```

Comment:
We have to decide what we want to measure. Those variables do not NEED to backtrack correctly and could be allocated in `global` which is way more efficient in JS.